src/en/02_devenv.md

```diff
@@ -103,14 +103,32 @@ It has a fundamental role in most Rust development:
 - It’s also a front-end to run complementary tools such as those that are
   described below, in the form of sub-commands.
 
-> **Warning**
+Cargo enables automatic dependencies resolution before compilation 
+by checking their checksums.
+File `Cargo.lock` contains all dependencies checksums which are compared to
```

Comment:
The other major benefit is that when an update of a dependency breaks things, it doesn't automatically break your CI. It does mean that you will have to manually update dependencies every once in a while though.

Reply:
Also maybe linking to https://doc.rust-lang.org/cargo/faq.html#why-have-cargolock-in-version-control would make sense?